Makefile

```
@@ -55,8 +56,19 @@ endif
 push_translations: ## push source strings to Transifex for translation
 	i18n_tool transifex push
 
-pull_translations:  ## pull translations from Transifex
+pull_xblock_translations:
```

```suggestion
pull_xblock_translations: ## pull xblock translations via atlas
```

This comment becomes the help text for `make help` so it's useful to have something.